app/main.ipynb

```diff
@@ -192,18 +244,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "A.mean(axis=1)",
```

Comment:
The task requires finding the mean for each column by the 'x' axis, which means you should use `axis=0` instead of `axis=1`.